includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -120,7 +120,6 @@ protected function buildTemplatePath( $controllerClass, $methodName, $forceRebui
 			$dirName = $this->getTemplateDir( $controllerClass );
 			foreach ( $templates as $templateName ) {
```

Comment:
I know that this code is not a part of the change but... it looks like it's executed quite frequently and scans through file system are not optimal, performance-wise.